src/Calendar/Calendar.test.js

```diff
@@ -412,6 +414,87 @@ describe('<Calendar />', () => {
         expect(wrapper.state('arrSelectedDates').length).toEqual(2);
     });
 
+    describe('weekdayStart', () => {
```

Comment:
Hopefully these tests make sense but I'm trying to verify that we are starting on the right weekday and that the right corresponds to the proper weekday.